src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -202,6 +201,11 @@ public async Task<ConflictResolution> ResolveConflictsAsync()
                             await conflictResolution.RemoveAllRenameAnnotationsAsync(documentsByProject, _renameAnnotations, _cancellationToken).ConfigureAwait(false);
                         }
                     }
+                    //
```

Comment:
Blank lines around this edit seem odd. No blank line before, double blank line after.

Reply:
I'll be a little more consistent. I was trying to separate contextual line comments and more "block" style comments without using `/* */` as the visual cue.